src/main/java/org/opensearch/flowframework/exception/FlowFrameworkException.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ *
+ * The OpenSearch Contributors require contributions made to
+ * this file be licensed under the Apache-2.0 license or a
+ * compatible open source license.
+ */
+package org.opensearch.flowframework.exception;
+
+public class FlowFrameworkException extends RuntimeException {
+    /**
+     * Constructor with error message.
+     *
+     * @param message message of the exception
+     */
+    public FlowFrameworkException(String message) {
```

Comment:
I like the idea of having our own custom exceptions.  Given that these will eventually need to be processed in a RestResponse, can we consider including the appropriate return code as part of constructing this exception?  For example, "index not found" should return `RestStatus.NOT_FOUND` etc.